src/rust/engine/fs/fs_util/src/main.rs

```diff
@@ -450,7 +450,6 @@ fn execute(top_match: &clap::ArgMatches<'_>) -> Result<(), ExitError> {
                 .unwrap()
                 .map(str::to_string)
                 .collect::<Vec<String>>(),
-              &[],
```

Comment:
Why does this compile both before and after this change? Not sure what I'm missing.

Reply:
For the record: it's because we removed a field of `PathGlobs` and edited the `create()` method elsewhere in this PR!